client/trino-cli/pom.xml

```diff
@@ -147,12 +147,6 @@
             <artifactId>junit-jupiter-engine</artifactId>
             <scope>test</scope>
         </dependency>
-
-        <dependency>
-            <groupId>org.testng</groupId>
-            <artifactId>testng</artifactId>
```

Comment:
why wasn't enforcer complaining that we have unused dependency declared?

Reply:
i think it doesn't enforce test scope at all